stairway/src/main/java/bio/terra/stairway/impl/FlightContextImpl.java

```diff
@@ -280,8 +309,11 @@ boolean isDoingLastStep() {
     return (isDoing() && getStepIndex() == steps.size() - 1);
   }
 
-  void nextStepIndex() {
+  void nextStepIndex() throws InterruptedException {
     logState.nextStepIndex();
+    // Update the stairway step progress meter to reflect what we step we are working on
+    setProgressMeter(
+        ProgressMetersImpl.STAIRWAY_STEP_PROGRESS, getStepIndex(), getStepClassNames().size());
```

Comment:
Very much a nit, but I see at least one other place where we infer total number of steps from `stepClassNames.size()`.  Would it be worth wrapping in a `getStepCount()` for readability?

Reply:
I did that. There were many places using `size()` either on the step or stepClassName list.